content/installation/setup/auth/Configure-http-header.md

```diff
@@ -0,0 +1,13 @@
+<!--
+title: "Configure HTTP Header"
+description: "Configure Contrast to connect to HTTP Header"
+tags: "installation setup EOP system settings connect http header "
+-->
+Contrast supports certificate login through a proxy using a HTTP Header token. To enable HTTP Header login: 
+
+* Update the property file by changing the ```authentication.mode``` in *~/path_to_contrast_installation/conf/auth.properties* to ```http_header```. 
+* By default, the HTTP Header key is ```Contrast-Authentication```. This is configurable in the ```teamserver_preferences``` table. You can update it with ```update teamserver_preferences set property_value='your_value_here' where property_name='authentication.http.header';```. 
```

Comment:
i've moved it into a property file. This little paragraph can be: 

"This is configurable in the auth.properties file if you update the value of `auth.http.header.field.name` "